<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio Settings</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Channels</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_2">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <item>
       <property name="text">
        <string>Mono</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Stereo</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Bit Depth</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_3">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <item>
       <property name="text">
        <string>16-bit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>24-bit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32-bit</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Hz</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="currentIndex">
       <number>5</number>
      </property>
      <item>
       <property name="text">
        <string>8000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11025</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22050</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>44100</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>48000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>88200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>96000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>176400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>192000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>352800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>384000</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox_2</tabstop>
  <tabstop>comboBox_3</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
